Integrates SCG Army into code; removes mention of Marines

FearTheBlackout opened the pull request:
:cl: Devildabeast
tweak: All Marine Corps uniforms and items have been renamed to reflect the SCG Army, removing all references of the Marines.
rscadd: Adds an array of Army outfits, spawnable by admins.
tweak: The Marine (now Army) PT uniform is now black instead of green.
/:cl:

No uniform changes beyond the PT gear. Army ranks currently reflect the ranks of the US Army as a placeholder until official ranks are designated for them.

MistakeNot4892 commented:
What is the point of adding an unusable placeholder branch?

ghost commented:
It's for admin events, although I'm not sure it actually has any effect if it's not spawnable. I can remove it if needed.

Nerev4r commented:
>adding event-only things

ghost commented:
What's wrong with that? It's better than leaving unused code and sprites lying around.

FreePrivateer commented:
I'd argue against this, as it would kinda remove marines from lore completely, and the marine uniforms, at least dress uniforms, are /quite visibly/ based on the USMC, but spaced up a a bit. Fatigues are fatigues, but might want to at least color swap some of the dress uniforms or something.

ghost commented:
Marines are already completely removed from lore, so that's not an issue.

I could easily recolor some of the uniforms if requested.

comma commented on maps/torch/torch_ranks.dm:
```diff
@@ -7,7 +7,8 @@
 		/datum/mil_branch/civilian,
-		/datum/mil_branch/solgov
+		/datum/mil_branch/solgov,
+		/datum/mil_branch/army
```
wouldn't this make it pickable in chargen?

comma commented on maps/torch/job/outfits.dm:
```diff
@@ -685,3 +685,143 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
+	back = null
+	shoes = /obj/item/clothing/shoes/dress
+
+/decl/hierarchy/outfit/job/torch/armyservice/medical
```
This and outfits below it are bloat, they're not joinable for departments, just base one and service are enough.

Anton-Kr commented:
litterally nothing wrong with having the assets there since the sprites are already made. I can use them for some away maps or something.

Sbotkin commented:
Branches exist solely for character creation code. If you need them for events, you can just add spawnable clothes.

afterthought2 commented:
Just remove them outright; renaming one unused branch to another is absurd. If people need the assets later, they can use git.

Anton-Kr commented:
As long as assets remain, I dont really care either way.

babydoll commented on maps/torch/items/clothing/solgov-accessory.dm:
```diff
@@ -635,103 +635,103 @@ ranks - fleet
+/obj/item/clothing/accessory/solgov/rank/army/enlisted/e7
 	name = "ranks (E-7 gunnery sergeant)"
-	desc = "Insignia denoting the rank of Gunnery Sergeant."
+	desc = "Insignia denoting the rank of Sergeant First Class."
```
The name doesn't agree

babydoll commented on maps/torch/items/clothing/solgov-accessory.dm:
```diff
@@ -635,103 +635,103 @@ ranks - fleet
\ No newline at end of file
+/obj/item/clothing/accessory/solgov/rank/army/flag/o10_alt
+	name = "ranks (O-10 general of the Army)"
+	desc = "Insignia denoting the rank of General of the Army."
```
Use the much more common 'Field Marshal' instead

FearTheBlackout replied:
Field Marshal of the Army or just Field Marshal?

babydoll commented on maps/torch/torch_define.dm:
```diff
@@ -12,7 +12,7 @@
 	overmap_size = 35
 	overmap_event_areas = 34
-	usable_email_tlds = list("torch.ec.scg", "torch.fleet.mil", "torch.marine.mil", "freemail.nt")
+	usable_email_tlds = list("torch.ec.scg", "torch.fleet.mil", "army.mil", "freemail.nt", "torch.scg")
```
Who is using army.mil and torch.scg?

Commit:
Removes Army branch; removes departmental Army service outfits
Removes Army branch; removes departmental Army service outfits


Fixes snow planet ore maps

They were so poor they didn't pass sanity. Now sanity is more tweakable.

Also fixes lava planet's ultra rich ore map actually being super poor.

Commit:
Integrates SCG Army into code; removes mention of Marines
